test/integration/search-bar/search-bar.test.ts

```diff
@@ -0,0 +1,86 @@
+import { assert, expect, test, describe } from "vitest";
+import SearchBar from "$lib/components/composites/search-bar/SearchBar.svelte";
+import { render, screen } from "@testing-library/svelte";
+import userEvent from "@testing-library/user-event";
+
+describe("SearchBarComponent", () => {
+    test("When all required props are provided, then the search bar does not throw any error", () => {
+        render(SearchBar, {
+            props: { onSearch: (input: string) => console.log(input) },
+        });
+        assert.doesNotThrow(() => screen.getByPlaceholderText("Search"));
+        expect(screen.getByPlaceholderText("Search")).toBeInTheDocument();
```

Comment:
These assertions do the same things, don't they?

Reply:
Good catch, I will delete one.